V5_MIGRATION.md

```diff
@@ -16,23 +16,58 @@ v5 introduces limited support for Swift Package Manager. See the [README](/READM
 
 In v4, 3D Secure classes were housed in the `BraintreePaymentFlow` module. In v5, `BraintreeThreeDSecure` is a standalone module offering the same 3DS functionality. The `BraintreePaymentFlow` module still houses Local Payment functionality.
 
-### CocoaPods
+#### Integration
+
+<details><summary>CocoaPods</summary>
+<p>
+
 In your Podfile, add:
 ```
 pod `Braintree/ThreeDSecure`
 ```
 
-### Carthage
+</p>
+</details>
+
+<details><summary>Carthage</summary>
+<p>
+
 You will need to add the `BraintreeThreeDSecure` framework to your project. See the Carthage docs for [integration instructions](https://github.com/Carthage/Carthage#adding-frameworks-to-an-application).
 
 *Note:* In v5, using the `--no-use-binaries` flag with `carthage update` may result in a timeout.
 
 *Note:* Long term support for Carthage is not guaranteed. Please update to SPM, if possible. Open a GitHub issue if there are concerns.
 
-### Swift Package Manager
+</p>
+</details>
+
+<details><summary>Swift Package Manager</summary>
+<p>
 
 The `BraintreeThreeDSecure` library does not currently support Swift Package Manager. It relies on a third party framework which we do not yet have in the `.xcframework` format.
 
+</p>
+</details>
+
+#### BTThreeDSecureRequestDelegate
+
+The signature for the `BTThreeDSecureRequestDelegate` method `onLookupComplete` method has changed:
```

Comment:
```suggestion
The signature for the `BTThreeDSecureRequestDelegate#onLookupComplete` method has changed:
```

Not sure if this is the right format for referencing a method in iOS docs, but wanted to remove the duplicate word.

Reply:
Ah, yeah it's kind of redundant as-is. Maybe we could just remove the second "method"?

The signature for the `BTThreeDSecureRequestDelegate` method `onLookupComplete` has changed:

Reply:
Yeah I think that reads better

Reply:
```suggestion
The signature for the `BTThreeDSecureRequestDelegate` method `onLookupComplete` has changed:
```